src/EnergyPlus/DaylightingManager.cc

```diff
@@ -679,120 +658,114 @@ void CalcDayltgCoefficients(EnergyPlusData &state)
         state.dataDaylightingManager->CreateDFSReportFile = false;
     }
 
-    for (ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
-        if (state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins == 0) continue;
+    // TODO MJW: For now preserve order by zone
```

Comment:
Why would the order be changed? I am thinking the same thing with SysAvailManagers.

Reply:
> Why would the order be changed? I am thinking the same thing with SysAvailManagers.

When everything for daylighting was done by zone, all of the reporting was by zone. Now, this should just loop over the daylighting controls objects, but those could be in a different order than the zones. With so many changes to daylighting, I didn't want a bunch of eio diffs that were simply order changes.